Gazebo fails to compile against graphviz 10.0

## Environment
* OS Version: macOS 
* Source or binary build? source build of 11.14.0

[![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gazebo-install-one_liner-homebrew-amd64&build=3068)](https://build.osrfoundation.org/job/gazebo-install-one_liner-homebrew-amd64/3068/) https://build.osrfoundation.org/job/gazebo-install-one_liner-homebrew-amd64/3068/

## Description
* Expected behavior: it should compile successfully with graphviz
* Actual behavior: it fails to compile

## Steps to reproduce
<!-- Provide steps so we can try to reproduce this issue -->

1. `brew install gazebo11`
2.
3.

## Output

The log in that CI job linked above does not include the relevant error, but I found the following when reproducing locally:

~~~
/tmp/gazebo11-20240220-52120-92epuq/gazebo-11.14.0/gazebo/gui/qgv/QGVSubGraph.cpp:47:53: error: use of undeclared identifier 'TRUE'
    Agnode_t *node = agnode(_sgraph->graph(), NULL, TRUE);
                                                    ^
/tmp/gazebo11-20240220-52120-92epuq/gazebo-11.14.0/gazebo/gui/qgv/QGVSubGraph.cpp:53:39: error: use of undeclared identifier 'TRUE'
    agsubnode(_sgraph->graph(), node, TRUE);
                                      ^
/tmp/gazebo11-20240220-52120-92epuq/gazebo-11.14.0/gazebo/gui/qgv/QGVSubGraph.cpp:69:56: error: use of undeclared identifier 'TRUE'
            ("cluster_" + _name).toLocal8Bit().data(), TRUE);
                                                       ^
/tmp/gazebo11-20240220-52120-92epuq/gazebo-11.14.0/gazebo/gui/qgv/QGVSubGraph.cpp:72:71: error: use of undeclared identifier 'TRUE'
        sgraph = agsubg(_sgraph->graph(), _name.toLocal8Bit().data(), TRUE);
                                                                      ^
make[2]: *** [gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/QGVSubGraph.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
~~~

Attempting to reproduce here:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gazebo-ci-gazebo11-homebrew-amd64&build=198)](https://build.osrfoundation.org/job/gazebo-ci-gazebo11-homebrew-amd64/198/) https://build.osrfoundation.org/job/gazebo-ci-gazebo11-homebrew-amd64/198/

I am testing a patch (https://github.com/gazebosim/gazebo-classic/commit/660ae15f4af1f5ea8d9d50d4a24e4d91a94f9c2d) to fix this.

--- a/gazebo/gui/qgv/QGVSubGraph.cpp
+++ b/gazebo/gui/qgv/QGVSubGraph.cpp
@@ -44,13 +44,13 @@ QString QGVSubGraph::name() const
 
 QGVNode *QGVSubGraph::addNode(const QString &label)
 {
-    Agnode_t *node = agnode(_sgraph->graph(), NULL, TRUE);
+    Agnode_t *node = agnode(_sgraph->graph(), NULL, 1);
     if (node == NULL)
     {
         qWarning()<<"Invalid sub node :"<<label;
         return 0;
     }
-    agsubnode(_sgraph->graph(), node, TRUE);
+    agsubnode(_sgraph->graph(), node, 1);
 
     QGVNode *item = new QGVNode(new QGVNodePrivate(node), _scene);
     item->setLabel(label);
@@ -66,10 +66,10 @@ QGVSubGraph *QGVSubGraph::addSubGraph(const QString &_name, bool cluster)
     if (cluster)
     {
         sgraph = agsubg(_sgraph->graph(),
-            ("cluster_" + _name).toLocal8Bit().data(), TRUE);
+            ("cluster_" + _name).toLocal8Bit().data(), 1);
     }
     else
-        sgraph = agsubg(_sgraph->graph(), _name.toLocal8Bit().data(), TRUE);
+        sgraph = agsubg(_sgraph->graph(), _name.toLocal8Bit().data(), 1);
 
     if (sgraph == NULL)
     {

